### ISSUE TITLE
DiscordSRV Errors

### ISSUE THREAD

**Issue by GamerCoder215** [Revxrsal/Lamp#85]
Hello,
One of my users using DiscordSRV is reporting some errors. I have the library shaded, so there shouldn't be any conflicts. Do you think there's an issue on your end or DiscordSRV's end?
```
[13:58:15 ERROR]: Could not pass event UnknownCommandEvent to StarCosmetics v1.3.2
java.util.NoSuchElementException: null
at java.util.LinkedList.getFirst(Unknown Source) ~[?:?]
at me.gamercoder215.starcosmetics.shaded.lamp.bukkit.brigadier.PaperCommodore$UnknownCommandListener.onUnknownCommand(PaperCommodore.java:75) ~[starcosmetics-1.3.2.jar:?]
at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor91.execute(Unknown Source) ~[?:?]
at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:81) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:git-Paper-409]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.20.4.jar:git-Paper-409]
at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[paper-1.20.4.jar:git-Paper-409]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
at org.bukkit.craftbukkit.v1_20_R3.CraftServer.dispatchCommand(CraftServer.java:995) ~[paper-1.20.4.jar:git-Paper-409]
at github.scarsz.discordsrv.listeners.DiscordConsoleListener.lambda$onGuildMessageReceived$0(DiscordConsoleListener.java:91) ~[DiscordSRV-Build-1.27.0.jar:?]
at github.scarsz.discordsrv.util.SchedulerUtil.lambda$runTask$0(SchedulerUtil.java:86) ~[DiscordSRV-Build-1.27.0.jar:?]
at io.papermc.paper.threadedregions.scheduler.FoliaGlobalRegionScheduler$GlobalScheduledTask.run(FoliaGlobalRegionScheduler.java:179) ~[paper-1.20.4.jar:?]
at io.papermc.paper.threadedregions.scheduler.FoliaGlobalRegionScheduler.tick(FoliaGlobalRegionScheduler.java:37) ~[paper-1.20.4.jar:?]
at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1639) ~[paper-1.20.4.jar:git-Paper-409]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:446) ~[paper-1.20.4.jar:git-Paper-409]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1515) ~[paper-1.20.4.jar:git-Paper-409]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1216) ~[paper-1.20.4.jar:git-Paper-409]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[paper-1.20.4.jar:git-Paper-409]
at java.lang.Thread.run(Unknown Source) ~[?:?]
[13:58:15 INFO]: That command is not registered in our database.
```

**Comment by Wector11211** [comment]
See #82 #83
The problem happens on `/` input, which is fixed in Lamp 3.1.9

**Comment by Revxrsal** [comment]
Yes, that was a bug in Lamp 3.1.8. As said above, it has been fixed in the latest version.
